website/content/docs/k8s/observability/metrics.mdx

```diff
@@ -0,0 +1,115 @@
+---
+layout: docs
+page_title: Metrics
+sidebar_title: Metrics
+description: Metrics for Consul on Kubernetes
+---
+
+# Metrics
+
+Consul on Kubernetes integrates with Prometheus and Grafana to provide metrics for Consul Service Mesh, including: service metrics, sidecar proxy metrics, Consul agent metrics, and gateway metrics. Service metrics can also be seen in the Consul UI Topology Visualization view. This section documents how to enable each of these.
+
+**Note:** Metrics will be supported in Consul-helm `0.31.0` and Consul-k8s `0.25.0`. However, enabling the [metrics merging feature](#connect-service-and-sidecar-metrics-with-metrics-merging) with Helm value (`defaultEnableMerging`) or annotation (`consul.hashicorp.com/enable-metrics-merging`) can only be used with Consul `1.10.0-alpha1` and above. The other metrics configuration can still be used before Consul `1.10.0-alpha1`.
+
+## Connect Service and Sidecar Metrics with Metrics Merging
```

Comment:
Before we get to the nitty gritty, what about a "quickstart" section that quickly shows the fast getting-started config with prometheus and grafana?

Reply:
Also, what do you think about having two sections: one for users that *don't* have services with their own prometheus metrics and one for users that *do*:

---

## Sidecar Metrics

If your services don't expose their own Prometheus metrics then you can safely enable sidecar metrics:

```yaml
metrics:
  enabled: true
```

After installation, you will be able to view sidecar metrics in Prometheus and Consul's UI.

## Sidecar Metrics Merging

If your services **do** expose their own Prometheus metrics, then you will need to enable metrics merging. Metrics merging is a feature that enables Prometheus to scrape both service and sidecar metrics from the same port. This is required because the Prometheus pod annotations do not support scraping from two ports.
..

Reply:
ooh I like both of these suggestions!